lib/demo_web/router.ex

```diff
@@ -1,12 +1,15 @@
 defmodule DemoWeb.Router do
   use DemoWeb, :router
 
+  import DemoWeb.UserAuth
+
   pipeline :browser do
     plug :accepts, ["html"]
     plug :fetch_session
     plug :fetch_flash
     plug :protect_from_forgery
```

Comment:
If a user leaves a form open in a browser for too long, and then his/her session expires unknowingly to the user, when that user finally submits the form, this blows up with `Plug.CSRFProtection.InvalidCSRFTokenError`. Would it be better to use `with: :clear_session` or to somehow intercept the raise and handle it more gracefully and user friendly?